src/dnddiceroller/Die.java

```diff
@@ -8,5 +10,21 @@
  * @author Paul Scarrone
  */
 public class Die {
-  
+    
+    int sides;
+    int value;
+
+    public Die(int sides) {
+        this.sides = sides;
+    }
+
+    public int getValue() {
+        return value;
+    }
+    
+    //add one to offset the starting point at zero
+    public void roll() {
+        value = (new Random()).nextInt(this.sides) + 1; 
```

Comment:
@ninjapanzer thank you for the feedback, I created a new commit with the changes.